CreatubblesAPIClient/Sources/Core/Data/NewCreationData.swift

```diff
@@ -52,6 +53,7 @@ open class NewCreationData: NSObject {
     open var creatorIds: Array<String>?
     open var creationYear: Int?
     open var creationMonth: Int?
+    open var asset: AVURLAsset?
```

Comment:
Hmmm, I understand it's required by the core app changes, it just doesn't feel completely right to put such a dependency in the API client, especially considering recent considerations in regards to `CreationDecorator` and some of its implementation details. Maybe in the future, we can think of another approach.